### PR TITLE
Updated to GraalVM 20.1.0.

### PR DESCRIPTION
**Pull request by tzezula**
1. Images for GraalVM native-images updated to use GraalVM 20.1.
2. Make the build of `images/build-native` faster by downloading the JVMCI JDK instead of building it.
3. Added images for JDK-11.

### CODE REVIEW

File: images/build-native/Dockerfile [hunk 1]
```diff
@@ -1,29 +1,24 @@
-FROM openjdk:8-jdk-slim as build
+FROM debian:buster-slim as build
```
Comment (mantree) [inline images/build-native/Dockerfile:1]: This looks a little out of place. Why the the move to buster?

Comment (tzezula) [reply images/build-native/Dockerfile:1]: It's a base image for the original `openjdk:8-jdk-slim`, see [openjdk:8-jdk-slim Dockerfile](https://github.com/docker-library/openjdk/blob/master/8/jdk/slim/Dockerfile).
We don't need image with JDK installed anymore as we don't build the JVMCI enabled JDK but we rather download it.

File: images/build-native/Dockerfile [hunk 2]
```diff
@@ -1,29 +1,24 @@
...
-
-RUN git clone https://github.com/oracle/graal.git \
-    && git -C graal checkout vm-19.2.0
+    && git clone --depth 1 https://github.com/graalvm/mx.git \
+    && wget -q --output-document jvmcijdk.tar.gz https://github.com/graalvm/openjdk8-jvmci-builder/releases/download/jvmci-20.1-b01/openjdk-8u242-jvmci-20.1-b01-linux-amd64.tar.gz \
+    && mkdir jvmcijdk8 \
+    && tar -xz --strip 1 -f jvmcijdk.tar.gz -C /build/jvmcijdk8 \
+    && rm jvmcijdk.tar.gz
+
+RUN git clone --single-branch https://github.com/oracle/graal.git --branch release/graal-vm/20.0
```
Comment (riconnon) [inline images/build-native/Dockerfile:18]: Could this have `--depth 1` to minimise download?

File: images/build-native/Dockerfile-jdk11 [hunk 3]
```diff
@@ -0,0 +1,40 @@
+FROM debian:buster-slim as build
+LABEL maintainer="tomas.zezula@oracle.com"
```
Comment (riconnon) [inline images/build-native/Dockerfile-jdk11:2]: Should this maintainer be a group rather than a user?

File: images/build-native/Dockerfile-jdk11 [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+WORKDIR /build
+
+RUN set -x \
+    && git clone --depth 1 https://github.com/graalvm/mx.git \
+    && wget -q --output-document jvmcijdk.tar.gz https://github.com/graalvm/labs-openjdk-11/releases/download/jvmci-20.0-b02/labsjdk-ce-11.0.6+9-jvmci-20.0-b02-linux-amd64.tar.gz \
+    && mkdir jvmcijdk11 \
+    && tar -xz --strip 1 -f jvmcijdk.tar.gz -C /build/jvmcijdk11 \
+    && rm jvmcijdk.tar.gz
+
+RUN git clone --single-branch https://github.com/oracle/graal.git --branch release/graal-vm/20.0
```
Comment (riconnon) [inline images/build-native/Dockerfile-jdk11:18]: Can we use `--depth 1` here?

File: images/init-native/docker-build.sh [hunk 5]
```diff
@@ -3,4 +3,11 @@ if [ -z "${FN_FDK_VERSION}" ];  then
     FN_FDK_VERSION=$(cat ../../release.version)
 fi
 sed -i.bak -e "s|<fdk\\.version>.*</fdk\\.version>|<fdk.version>${FN_FDK_VERSION}</fdk.version>|" pom.xml && rm pom.xml.bak
+cp Dockerfile Dockerfile.build
 docker build -t fnproject/fn-java-native-init:${FN_FDK_VERSION} -f Dockerfile-init-image .
+
+cp Dockerfile Dockerfile.build
+sed -i.bak -e "s|fnproject/fn-java-fdk-build:latest|fnproject/fn-java-fdk-build:jdk11-latest|" Dockerfile.build && rm Dockerfile.build.bak
```
Comment (riconnon) [inline images/init-native/docker-build.sh:10]: This naming is confusing... `latest` is going to mean JDK 8 which is definitely not the latest!
I think, however, this is consistent with the "main" Java FDK so I'll punt.
Hopefully it's something we can revisit "soon"

File: images/build-native/Dockerfile-jdk11 (hunk 4 above)
Comment (tzezula) [reply images/build-native/Dockerfile-jdk11:18]: fixed.

Comment (tzezula) [reply images/build-native/Dockerfile-jdk11:18]: fixed.

File: images/build-native/Dockerfile (hunk 2 above)
Comment (tzezula) [reply images/build-native/Dockerfile:18]: fixed.

File: images/build-native/Dockerfile-jdk11 (hunk 3 above)
Comment (tzezula) [reply images/build-native/Dockerfile-jdk11:2]: Fixed.

File: images/init-native/docker-build.sh (hunk 5 above)
Comment (tzezula) [reply images/init-native/docker-build.sh:10]: Yes, the naming is the same as in "main" Java FDK.

**Pull request merged**